packages/clerk-js/src/core/resources/Organization.ts

```diff
@@ -155,16 +155,18 @@ export class Organization extends BaseResource implements OrganizationResource {
     }).then(res => new Organization(res?.response as OrganizationJSON));
   };
 
-  protected fromJSON(data: OrganizationJSON): this {
-    this.id = data.id;
-    this.name = data.name;
-    this.slug = data.slug;
-    this.logoUrl = data.logo_url;
-    this.publicMetadata = data.public_metadata;
-    this.membersCount = data.members_count;
-    this.pendingInvitationsCount = data.pending_invitations_count;
-    this.createdAt = unixEpochToDate(data.created_at);
-    this.updatedAt = unixEpochToDate(data.updated_at);
+  protected fromJSON(data: OrganizationJSON | null): this {
+    if (data) {
```

Comment:
**This is an extremely nitpicky comment and there's absolutely no need to address it**
I usually like to reverse if statements like that and use them as guard expressions, eg:

```tsx
if (!data) {
 return this;
} 
...
```

as it makes the edge case stand out more